src/hooks/useAccountTotalStaked.js

```diff
@@ -1,10 +1,10 @@
 import { useMemo } from 'react'
-import { useAccountStakes } from './useStakes'
+import { useAccountStakesByGarden } from './useStakes'
 
 import BigNumber from '../lib/bigNumber'
 
 export default function useAccountTotalStaked(account) {
-  const stakes = useAccountStakes(account)
+  const stakes = useAccountStakesByGarden(account)
```

Comment:
we don't need to pass the account  since we can use the useWallet inside the hook, there are several places were we are calling it with the account

Reply:
ah yeah i.e the profile that sometimes we get the address from the url rihgt? if that the case makes sense yes